authors/apps/authentication/tests/test_reset_password.py

```diff
@@ -22,7 +22,6 @@ def test_invalid_email(self):
         '''Test invalid email'''
         response = self.client.post(self.FORGOT_URL, {"email": "qw"}, format="json")
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-        self.assertEqual(json.loads(response.content)["email"][0], "Enter a valid email address.")
 
     def test_unregistered_user(self):
         '''Test unregister user'''
```

Comment:
The codebase has a mixture of single-quotes and double quotes for the same kind of strings. Consider sticking to a standard (https://www.python.org/dev/peps/pep-0008/#string-quotes). This is just an example